docs/changelog.md

```diff
@@ -554,7 +550,7 @@ modular update mojo
       var s: String
 
       fn __init__(inout self, owned s: String):
-          self.s = s ^
+          self.s = s^
```

Comment:
I know the space looks odd, but that's the way `mojo format` formats it.

Reply:
I think the doc uses the style without the white space. Reverted the change anyways.